### DISCUSSION TITLE
面板加载缓慢且部分模块无法加载

### DISCUSSION THREAD

**Discussion by Aurora-Dylan** [1Panel-dev/1Panel#5797] [Q&A]
### 联系方式

91unicorn@proton.me

### 1Panel 版本

1.10.11-lts

### 问题描述

服务器在东京，加载面板是加载缓慢，部分的js文件加载时间过长，应用商店无法正常访问
但是其中部署的服务能够正常的访问
![2024-07-14_09-21-38](https://github.com/user-attachments/assets/25db3de8-8839-43dd-a129-9eb720e9b8d1)

### 重现步骤

登录

**Comment by wanghe-fit2cloud** [comment]
感谢您的反馈！出现上述问题基本是由服务器网络问题引起的。应用商店资源也统一维护在阿里 OSS 中。我们会在后续版本中尝试进行优化。
